### PR TITLE
Added swedish locale

### LINKED ISSUE #52: Add swedish locale
Swedes are all about astrology.

### PR DESCRIPTION
**Pull request by peter-rymdfabriken**

### CODE REVIEW

**Comment by helmasaur** [comment]
Sorry for the delay… I'll look at it in the next days! 😁

**Comment by helmasaur** [comment]
I got really busy lately. I will update the npm package tomorrow!

File: locales/sv/zodiac.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+	"aries": {
+		"name": "Väduren",
+		"element": 0,
+		"stone": "blodsten"
```
Comment (helmasaur) [inline locales/sv/zodiac.json:5]: In Swedish, is it "blodsten" or "heliotrop"?

File: locales/sv/zodiac.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	},
+	"gemini": {
+		"name": "Tvillingarna",
+		"element": 2,
+		"stone": "agat"
+	},
+	"cancer": {
+		"name": "Kräftan",
+		"element": 3,
+		"stone": "emerald"
```
Comment (helmasaur) [inline locales/sv/zodiac.json:20]: In Swidish, is it "emerald" or "smaragd"?

**Review by helmasaur — CHANGES_REQUESTED** [review]
Thanks for the translation! I just have two comments before merging. In those two situations, are the words written the right ones? Thanks!

File: locales/sv/zodiac.json (hunk 2 above)
Comment (peter-rymdfabriken) [reply locales/sv/zodiac.json:20]: My bad, emerald is actually the english word. Correct swedish should be "Smaragd"

File: locales/sv/zodiac.json (hunk 1 above)
Comment (peter-rymdfabriken) [reply locales/sv/zodiac.json:5]: heliotrop and blodsten refers to the same kind of stone, so either or is fine.

**Review by helmasaur — APPROVED** [review]
Hey! First of all, I'm sorry for the time it took for me to answer.

Thanks for the changes you made! I approve them. Although, let me ask yo uif blodsten is more used than heliotrop or not. If it is, we can change it in a future version 😄.

**Pull request merged**